docs/roadmap/roadmap.md

```diff
@@ -94,4 +94,4 @@ Scope is still TBD.
 
 #### H2 April
 
-- [`v5.0.0`](https://github.com/cosmos/ibc-go/milestone/21): Final release that includes the update of Cosmos SDK from `v0.46` to [v1.0](https://github.com/cosmos/cosmos-sdk/milestone/52) and that will support the migration to SMT storage.
\ No newline at end of file
+- [`v5.0.0`](https://github.com/cosmos/ibc-go/milestone/21): Final release that includes the update of Cosmos SDK from `v0.46` to [v1.0](https://github.com/cosmos/cosmos-sdk/milestone/52) and that will support the migration to SMT storage.
```

Comment:
I believe you don't need to push any changes to the roadmap file, right?

Reply:
I think the code editor just adds a newline at the end of the file